packages/storybook/stories/components/HorizontalScroll.stories.tsx

```diff
@@ -12,15 +12,23 @@ import {
   SourceMemberRole,
   SourceType,
 } from '@dailydotdev/shared/src/graphql/sources';
-import {
-  UnFeaturedSquadCardProps
-} from '@dailydotdev/shared/src/components/cards/squad/common/types';
-
-
-const ScrollableElement = ({ item, index, ...props }: {
-  item: { value: string },
-  index: number
-}): ReactElement => <div {...props} className="w-full max-w-40 border p-4">{item.value} <br/>{index}</div>;
+import { UnFeaturedSquadCardProps } from '@dailydotdev/shared/src/components/cards/squad/common/types';
+import AuthContext from '@dailydotdev/shared/src/contexts/AuthContext';
+import { QueryClientProvider } from '../extension/_providers';
+
+const ScrollableElement = ({
+  item,
+  index,
+  ...props
+}: {
+  item: { value: string };
+  index: number;
+}): ReactElement => (
+  <div {...props} className="w-full max-w-40 border p-4">
```

Comment:
border won't do anything without color?

Reply:
solid black is what being used with just border. This is just for display anyway, hence made it look like it is not really part of the design system.

Reply:
Ah I see :) thanks for the explanation.